kolibri_explore_plugin/assets/src/views/CustomChannelPresentationApp.vue

```diff
@@ -1,27 +1,28 @@
 <template>
 
-  <iframe
-    ref="iframe"
-    class="custom-presentation-iframe"
-    frameBorder="0"
-    :src="rooturl"
-  >
-  </iframe>
+  <div>
```

Comment:
I don't see why this new div wapping the iframe is needed. Maybe for the scroll?

Reply:
It's not needed at all, it's there because at first I added the new `ContentModal` component inside this, instead of in the `ExploreIndex` one and I forget to remove the div wrapper.